data/css/widgets/_pDFView.scss

```diff
@@ -2,6 +2,11 @@
 
     background-color: $background-light-color;
 
+    /* Both panel and content */
+    scrollbar.vertical slider {
+        min-height: 50px;
```

Comment:
Adwaita has 40px as `min-height`, we should do the same. And also do this for horizontal sliders as well.